spec/system/admin_views_volunteers_page_spec.rb

```diff
@@ -26,6 +26,14 @@
       expect(page).to have_text("Miles Driven")
       expect(page).to have_text("Want reimbursement?")
     end
+
+    it "displays default logo" do
+      sign_in admin
+
+      visit volunteers_path
+
+      expect(page).to have_xpath("//img[@src = '/packs-test/media/src/images/default-logo-c9048fc43854499e952e4b62a505bf35.png' and @alt='CASA Logo']")
```

Comment:
I don't think that this hash will stay the same over time? But maybe it will? 
`'/packs-test/media/src/images/default-logo-c9048fc43854499e952e4b62a505bf35.png`

Reply:
I *think* this hash will stay the same since we're on webpacker v 5.2, which uses ```contenthash``` instead of ```chunkhash``` to generate this -- it should be 100% based on the file contents.

https://github.com/webpack/webpack.js.org/issues/2096

That said, I'm happy to update this to just look for 'default-logo' so the test is more resilient :)

Reply:
Let's leave it as is and if it breaks then we'll change it :)